assignment-2/src/view/dumb/AnswersList.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import MyNavbar from "./MyNavbar";
+import MyVoteSystem from "./MyVoteSystem";
+import MyCard from "./MyCard";
+
+const AnswersList = ({
+                         question, sOUsername, answers, newAnswer,
+                         onDeleteAnswer, onEditPress, onChangeNewAnswer, onEditSubmit, onViewAnswers,
+                         onChange, onCreate, onLogout, onVoteQuestion, onVoteAnswer
+                     }) => (
+    <div>
+        <MyNavbar searchBar={false} loggedInAsButton={true} logoutButton={true} sOUsername={sOUsername}
+                  onLogout={onLogout}/>
+
+        <br/>
+
+        <div className="container" style={{"padding-top": "5em"}}>
+
+            <div>
+                <div className={"card myCard"}>
+                    <div className="card-body">
+                        <div className="row">
+                            <div className="col-sm-1">
+                                <MyVoteSystem qAType={"question"}
+                                              sOUserName={sOUsername}
+                                              questionAnswer={question}
+                                              onVote={onVoteQuestion}/>
+                            </div>
+                            <div className="col-sm-11" style={{"padding-top": "1em"}}>
+                                <MyCard qAType="question"
+                                        questionAnswer={question}
+                                        onViewAnswers={onViewAnswers}/>
+                                <br/>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <br/>
+                <br/>
+            </div>
+
+            {
+                answers.sort(function (a1, a2) {
+                    return a2.score - a1.score;
+                }).map((answer, index) => (
+                    <div key={index}>
```

Comment:
This component is still a little too big :) For example you could have moved this answer card to its own dumb component.